libfwupdplugin/fu-bluez-device.c

```diff
@@ -405,8 +410,11 @@ fu_bluez_device_read_battery_interface(FuBluezDevice *self,
 	obj_percentage =
 	    fu_bluez_device_get_ble_property(obj_path, iface_name, "Percentage", error);
 	if (obj_percentage == NULL) {
-		g_prefix_error(error, "failed to get battery percentage from org.bluez.Battery1: ");
-		return FALSE;
+		g_debug("failed to get battery percentage from org.bluez.Battery1: %s",
+			(*error)->message);
+		g_clear_error(error);
```

Comment:
We don't enforce that @error can't be NULL -- you need to use error_local here I'm afraid.